socorro/processor/mozilla_transform_rules.py

```diff
@@ -1022,7 +1022,7 @@ def _get_pretty_os_version(self, processed_crash):
         major_version = int(version_split[0])
         minor_version = int(version_split[1])
 
-        if processed_crash.os_name == 'Windows':
+        if processed_crash.os_name.lower().startswith('windows'):
```

Comment:
It's the same thing (though as soon as I had pushed I realized I should have used ``processed_crash.os_name`` instead in the verification above. )